addons/account/views/account_view.xml

```diff
@@ -1010,22 +1040,22 @@
                                         <field name="credit"/>
                                         <field name="quantity"/>
                                     </group>
-                                    <group string="Accounting Documents">
-                                        <field name="move_id" required="False"/>
-                                        <field name="statement_id" readonly="True"/>
+                                    <group string="Accounting" invisible="[('move_id','=',False), ('statement_id','=',False)]">
```

Comment:
I'd keep the label "Documents" (because everytinh is about accounting in that view ^^

Also, 
* move_id cannot be empty, so the attrs makes no sense
* I'd add the invoice link there too